roster/src/components/NavBar.vue

```diff
@@ -1,6 +1,12 @@
 <template>
 	<div class="h-12 bg-white border-b px-12 flex items-center">
-		<a class="text-xl" href="/">Home</a>
+		<div class="flex items-center space-x-1.5">
+			<a href="/app/home" class="text-gray-600 hover:text-gray-700">Home</a>
+			<FeatherIcon name="chevron-right" class="h-4 w-4" />
+			<a href="/app/hr" class="text-gray-600 hover:text-gray-700">HR</a>
```

Comment:
Should we instead call it Frappe HR and have the logo too. Will feel like it's a part of the product.
Maybe we can drop **Home**